anaUltraScurve.py

```diff
@@ -208,18 +223,73 @@ def overlay_fit(VFAT, CHAN):
     vthr_list[event.vfatN][event.vfatCH] = event.vthr
     trim_list[event.vfatN][event.vfatCH] = event.trimDAC
     trimrange_list[event.vfatN][event.vfatCH] = event.trimRange
+    if options.SaveFile:
+        fitter.feed(event)
+        pass
     pass
 
 if options.SaveFile:
+    scanFits = fitter.fit()
+    pass
+
+# Determine hot channels
+import numpy as np
+if options.SaveFile:
+    print 'Determining hot channels'
     masks = []
+    maskReasons = []
```

Comment:
Let's define this container as a dict instead in `anaInfo.py` let's have the dictionary be (int, string) pairs where the key value is the int and the string is the human readable reason for being masked.  Then in the tree store the key value (e.g. the int).

I think this will be easier to maintain, read, and reference in the future.